requirements/dev-requirements.txt

```diff
@@ -10,15 +10,17 @@ alembic==1.4.3
     # via -r base-requirements.in
 amqp==2.6.1
     # via kombu
+appnope==0.1.2
+    # via ipython
 architect==0.5.6
     # via -r base-requirements.in
 attrs==18.2.0
     # via
     #   -r base-requirements.in
     #   jsonschema
-babel==2.8.0
+babel==2.9.1
     # via
-    #   django-phonenumber-field
+    #   -r base-requirements.in
```

Comment:
Why is `django-phonenumber-field` no longer listed? Is it possible `babel` is no longer a requirement?

Reply:
because I had to explicitly define the version in `base-requirements.in` (there is a comment next to it explaining why it was added), as it was still installing a version lower than `2.9.1` by following `django-phonenumber-field` alone

Reply:
It should list all dependents, including `django-phonenumber-field`, even if we require it explicitly. Example of another dependency that does that:

https://github.com/dimagi/commcare-hq/blob/dd42c35e5a7a9f007e7987fb9d9394bdc0dcdaf8/requirements/dev-requirements.txt#L83-L89

Reply:
Could it be that `babel` was removed as a dependency (or shifted to a test-only dependency)?

Reply:
got it. looks like it's still needed in dev, docs, and prod requirements, but otherwise looks like `django-phonenumber-field` has removed it as a requirement. updated to reflect this

Reply:
Thanks!